policyengine_us/variables/gov/states/ar/tax/income/credits/ar_inflation_relief_credit.py

```diff
@@ -0,0 +1,25 @@
+from policyengine_us.model_api import *
+
+
+class ar_inflation_relief_credit(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Arkansas inflation relief income-tax credit"
+    unit = USD
+    definition_period = YEAR
+    defined_for = StateCode.AR
+
+    def formula(tax_unit, period, parameters):
+        income = tax_unit("ar_taxable_income", period)
```

Comment:
Can you add a note about this being the relevant income concept? The parameter description mentions AGI. It says line 28 so just connect that